src/commands/contract/new.ts

```diff
@@ -96,13 +97,9 @@ export class NewContract extends SwankyCommand<typeof NewContract> {
     await ensureDir(path.resolve(projectPath, "tests", args.contractName));
 
     await this.spinner.runCommand(async () => {
-      this.swankyConfig.contracts[args.contractName] = {
-        name: args.contractName,
-        moduleName: snakeCase(args.contractName),
-        deployments: [],
-      };
-
-      await writeJSON(path.resolve("swanky.config.json"), this.swankyConfig, { spaces: 2 });
+      const configBuilder = new ConfigBuilder(getSwankyConfig("local"));
+      configBuilder.addContract(args.contractName);
```

Comment:
nit: it might be a bit more convenient to make `addContract`, `addContractDeployment` methods more suited for the builder pattern (to return `this`) - we could then chain all configuration into a single expression

Reply:
Indeed! Let me adjust it